courier/sms.go

```diff
@@ -29,7 +29,7 @@ func (c *courier) QueueSMS(ctx context.Context, t SMSTemplate) (uuid.UUID, error
 	message := &Message{
 		Status:       MessageStatusQueued,
 		Type:         MessageTypeSMS,
-		Channel:      "sms",
+		Channel:      "phone",
```

Comment:
Why use `"phone"`? IMO, this is not ideal as it is not clear whether the message is delivered as an SMS or phone call.

Reply:
I kinda agree with this, see discussion here: https://github.com/ory/kratos/pull/3649#discussion_r1436955299

Reply:
So you agree with sms or phone? I just thought that it makes more sense to make this as precise as possible, because it would be confusing to have later "phone" and "phone_call".

In the end no one forces you to use this for sms, but that's why we have the channels. One can add "phone_call", "whatsapp", "my_native_app_push_notification", or any other channel (once we fully expose that).

I am also not sure if we really need to mirror all the intermediary states of phone calls, all that Kratos cares about is whether the correct code was submitted, regardless how the user got the code.

Reply:
I thought that `sms` made the most sense

Reply:
Okay, I refactored the whole thing to use "sms" now.